packages/module/src/elements/BaseElement.ts

```diff
@@ -60,7 +59,7 @@ export default abstract class BaseElement<E extends ElementModel = ElementModel,
       controller: observable.ref,
       label: observable,
       style: observable,
-      ordering: computed({ equals: _.isEqual })
+      ordering: computed({ equals: (a, b) => JSON.stringify(a || {}) === JSON.stringify(b || {}) })
```

Comment:
This one concerns me a bit.
`ordering` is an array of numbers. Can we make a specific comparison function.